docs/data/routes/docs/nextjs/creating-components/new-component/en.md

```diff
@@ -0,0 +1,221 @@
+
+6. Open `/sitecore/content/MyProject/Home` in the Experience Editor and add your new rendering, including creating a data source item for it.
+
+   > Your rendering host page outputs `Unknown component` or a similar message because you have not yet mapped a component to this JSON rendering in your rendering host.
+
+7. Publish all your item changes:
+
+   - In Content Editor, on the **Publish** ribbon, click the small black arrow next to the **Publish** icon and click **Publish site**.
+   - In the **Publish Site** window, to publish your items from the Master database to the Web database, select the **Smart publish** radio button and click **Publish**.
+
+8. You can now test the output of your new rendering:
+
+   - In your rendering host, open the site home page.
+
+   - Type the following into a PowerShell terminal to get the rendering host logs: `docker-compose logs -f rendering`.
+
+   - In the rendering host logs, view the debug output with the Layout Service response, including the component you just added:
+
+     ```json
+     {
+         "uid": "ba5d4f2d-b6f1-428a-81e8-6b7c25844c08",
+         "componentName": "DataSourceExample",
+         "dataSource": "{A2A3F4C0-B13B-4651-B342-320E56FDC43A}",
+         "params": {},
+         "fields": {
+             "Title": {
+                 "value": "Default"
+             },
+             "BodyText": {
+                 "value": "Default"
+             }
+         }
+     }
+         
+     ```
+
+### Create the component in the Next.js app
+
+In the Next.js-based application, you must now create a component matching the rendering you just created.
+
+1. Create the file `\MyProject\src\rendering\components\DataSourceExample.tsx`.
+2. In the newly created file, define a component with the same name as the rendering:
+
+   ```typescript
+   import { Text, RichText, Field } from '@sitecore-jss/sitecore-jss-nextjs';
+
+   type DataSourceExampleProps = {
+       fields: {
+       	Title: Field<string>;
+       	BodyText: Field<string>;
+     	};
+   };
+   
+   const DataSourceExample = ({ fields }: DataSourceExampleProps): JSX.Element => (
+     <>
+       <Text tag="h2" className="display-4" field={fields.Title} />
+   
+       <RichText className="contentDescription" field={fields.BodyText} />
+     </>
+   );
+   
+   export default DataSourceExample;
+    
+   ```
+
+### Fill in values in the Experience Editor
+
+To fill in the values:
+
+1. In the Experience Editor, open `/sitecore/content/MyProject/home`.
+2. Fill in values for all fields.
+3. Click **Save** and **Publish**.
+4. Refresh `https://myproject.localhost` and see the changes.
+
+> Be aware of the following when you use the solution:
+>
+> * A publishing of the `Platform` project updates the running `cm` service.
+> * The running rendering service uses `dotnet watch` and re-compiles automatically for any changes you make. You can also run the `rendering` project directly from Visual Studio.
```

Comment:
Actually it uses `next dev` not `dotnet watch` to achieve this for Next.js. :) But I think this note is unnecessary actually, we can remove everything in this "Be aware" section if you are OK with it.